test/py/CMakeLists.txt

```diff
@@ -44,6 +66,9 @@ endfunction()
 add_dependencies(tests migraphx_py)
 add_dependencies(check migraphx_py)
 
+add_py_venv_fixture(
```

Comment:
This should be called inside `add_py_test`.

Reply:
Discussed this on Friday with Paul, and he suggested to check if a test already exists before adding tests for init/setup venv to resolve the issue with duplication of test names. I have now made that change and `add_py_venv_fixture` is being called inside `add_py_test`.